index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <title>My portfolio</title>
+  </head>
+  <body>
+    <section class="hero">
+      <header class="nav-container">
+        <a href="#" class="nav-logo">My Logo</a>
+        <nav class="nav-bar">
+          <ul class="menu-list">
+            <li class="menu-item"><a href="#">Portfolio</a></li>
+            <li class="menu-item"><a href="#">About</a></li>
+            <li class="menu-item"><a href="#">Contact</a></li>
+          </ul>
+          <div class="mob-menu">
+            <div class="mob-menu-bar"></div>
+            <div class="mob-menu-bar"></div>
+            <div class="mob-menu-bar"></div>
+          </div>
+        </nav>
+      </header>
+      <section class="bio-content">
+        <div class="bio-content-wrapper">
+          <h1>I’m Wilfried. <br />Glad to see you!</h1>
+          <div class="bio">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </div>
+          <a href="#">LET’S CONNECT</a>
```

Comment:
- According to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.), Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

**Original Design**
![headline-mobile](https://user-images.githubusercontent.com/97834160/235264905-ee12073b-1cae-4b06-8b84-51fee9fb622e.png)

**Your Design**
![Screenshot from 2023-04-29 00-30-55](https://user-images.githubusercontent.com/97834160/235264895-0ca0d2b9-762c-4e72-8d52-e4bcf9531a64.png)

Pay Close Attention to the Background Image and to the margins.